content/faq.md

```diff
@@ -1,9 +1,29 @@
-title: complate Frequently Asked Questions 
+title: complate Frequently Asked Questions
 h1: FAQ
-subtitle: Frequently asked questions
+subtitle: Frequently Asked Questions
 safe: false
 
 
+Is complate Right for Me? <span id="trade-offs"></span>
+-------------------------
+
+While complate has many advantages, it also introduces additional complexity:
+The use of JavaScript typically requires both a compiler (for module resolution and to transform JSX) and
+a JS runtime environment (typically provided via [the respective adaptor](getting-started.html)) --
+see [How It Works](how-it-works.html) for a detailed explanation of that interplay.
```

Comment:
This, too, is a forward reference: I wanna explain how complate works as a view layer, based on diagrams like this:

![complate](https://user-images.githubusercontent.com/3515/75383430-9b504580-58dc-11ea-9acb-9acde7e65989.png)